src/Service/PartyHelper.php

```diff
@@ -209,4 +209,38 @@ function ($relation) {
 
         return ['complainants' => $complainants, 'counterparties' => $counterparties];
     }
+
+    public function getSortingRelevantComplainant(CaseEntity $case): string
```

Comment:
I think this function should be called `getSortingComplainant` because that's what it does.

Reply:
Removed `Relevant` from function name.